src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -2776,6 +2776,256 @@
 short-summary: List out all available versions of Bicep CLI.
 """
 
+helps['stack'] = """
+type: group
+short-summary: (Version 1.9) 
+"""
+
+helps['stack mg create'] = """
+type: command
+short-summary: Create or update a deployment stack at management group scope
+examples:
+  - name: Create a deployment stack using template file.
+    text: az stack mg create --name "StackName" --management-group-id myMg --template-file simpleTemplate.json --location "westus2" --description "description"
```

Comment:
These quotation marks could be omitted. 
e.g., `az stack mg create --name StackName --management-group-id MyMg --template-file simpleTemplate.json --location westus2 --description MyDescription`.
Same below.